### ISSUE TITLE
Can't interact with code

### ISSUE THREAD

**Issue by StarixCrown** [freeCodeCamp/freeCodeCamp#14534]
Challenge [Create a Form Element](https://www.freecodecamp.com/challenges/create-a-form-element) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
<style>
  .red-text {
    color: red;
  }

  h2 {
    font-family: Lobster, Monospace;
  }

  p {
    font-size: 16px;
    font-family: Monospace;
  }

  .thick-green-border {
    border-color: green
    border-width: 10px;
    border-style: solid;
    border-radius: 50%;
  }

  .smaller-image {
    width: 100px;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p>Click here for <a href="#">cat photos</a>.</p>

<a href="#"><img class="smaller-image thick-green-border" alt="A cute orange cat lying on its back. " src="https://bit.ly/fcc-relaxing-cat"></a>

<p>Things cats love:</p>
<ul>
  <li>cat nip</li>
  <li>laser pointers</li>
  <li>lasagna</li>
</ul>
<p>Top 3 things cats hate:</p>
<ol>
  <li>flea treatment</li>
  <li>thunder</li>
  <li>other cats</li>
</ol>
<input type="text" placeholder="cat photo URL">
<form action="/sum"
  
```
I cant write anything in the code , I can write only after I press backspace once but then the page on the phone image crashes saying Chrome detected unusual code

**Comment by JoolsMcFly** [comment]
What happens if you hit Reset and start again with fresh code?
Also, make sure you text input is inside a form.

**Comment by QuincyLarson** [comment]
@StarixCrown Closing as a duplicate of  https://github.com/freeCodeCamp/freeCodeCamp/issues/13727.
